compiler/bcq-tools/generate_bcq_output_arrays

```diff
@@ -41,6 +41,18 @@ def _get_parser():
         type=str,
         help="Full filepath of the output file.",
         required=True)
+    parser.add_argument(
+        "-m",
+        "--metadata_path",
+        type=str,
+        help="Full filepath for the file that provides metadata.",
+        required=True)
+    parser.add_argument(
+        "-O",
```

Comment:
One more minor thing.
We used `-O` for output file path.
As this is for input file that provides output arrays (I think), how about using `-A` ?